cts/cli/regression.tools.exp

```diff
@@ -23,6 +23,463 @@ A new shadow instance was created. To begin using it, enter the following into y
 </cib>
 =#=#=#= End test: Validate CIB - OK (0) =#=#=#=
 * Passed: cibadmin       - Validate CIB
+=#=#=#= Begin test: List non-advanced cluster options =#=#=#=
+Pacemaker cluster options
+
+Also known as properties, these are options that affect behavior across the entire cluster. They are configured within cluster_property_set elements inside the crm_config subsection of the CIB configuration section.
+
+  * dc-version: Pacemaker version on cluster node elected Designated Controller (DC)
+    * Includes a hash which identifies the exact revision the code was built from. Used for diagnostic purposes.
+    * Possible values (generated by Pacemaker): string (default: )
+
+  * cluster-infrastructure: The messaging layer on which Pacemaker is currently running
+    * Used for informational and diagnostic purposes.
+    * Possible values (generated by Pacemaker): string (default: )
+
+  * cluster-name: An arbitrary name for the cluster
+    * This optional value is mostly for users' convenience as desired in administration, but may also be used in Pacemaker configuration rules via the #cluster-name node attribute, and by higher-level tools and resource agents.
+    * Possible values: string (no default)
+
+  * dc-deadtime: How long to wait for a response from other nodes during start-up
+    * The optimal value will depend on the speed and load of your network and the type of switches used.
+    * Possible values: time (default: )
+
+  * cluster-recheck-interval: Polling interval to recheck cluster state and evaluate rules with date specifications
+    * Pacemaker is primarily event-driven, and looks ahead to know when to recheck cluster state for failure-timeout settings and most time-based rules. However, it will also recheck the cluster after this amount of inactivity, to evaluate rules with date specifications and serve as a fail-safe for certain types of scheduler bugs. A value of 0 disables polling. A positive value sets an interval in seconds, unless other units are specified (for example, "5min").
+    * Possible values: time (default: )
+
+  * fence-reaction: How a cluster node should react if notified of its own fencing
+    * A cluster node may receive notification of a "succeeded" fencing that targeted it if fencing is misconfigured, or if fabric fencing is in use that doesn't cut cluster communication. Use "stop" to attempt to immediately stop Pacemaker and stay stopped, or "panic" to attempt to immediately reboot the local node, falling back to stop on failure.
+    * Possible values: "stop" (default), "panic"
+
+  * election-timeout: Declare an election failed if it is not decided within this much time. If you need to adjust this value, it probably indicates the presence of a bug.
```

Comment:
Well, these aren't being filtered out like they should be. Neither is the deprecated option (`remove-after-stop`). They're not even separated unless `--all` is given.